### DISCUSSION TITLE
V2.0rc，not allowed

### DISCUSSION THREAD

**Discussion by 1424784560** [tauri-apps/tauri#10757] [Q&A]
Using 2.0rc, calling API, permission issue，such as setting titles

![Uploading 1724483008501.jpg…]()

**Comment by 1424784560** [comment]
![1724483008501](https://github.com/user-attachments/assets/788845f1-2cdf-4309-8d95-f5a7921ad440)

**Comment by StepanZagray** [comment]
In [docs](https://v2.tauri.app/plugin/window-customization/#permissions) it says that you need to set those permissions
